shotover-proxy/src/transforms/mod.rs

```diff
@@ -369,6 +404,27 @@ impl<'a> Wrapper<'a> {
         result
     }
 
+    pub async fn call_next_transform_pushed(mut self) -> ChainResponse {
+        let transform = match self.transforms.next() {
+            Some(transform) => transform,
+            None => return Ok(self.messages),
+        };
+
+        let transform_name = transform.get_name();
+        let chain_name = self.chain_name.clone();
+
+        let start = Instant::now();
+        let result = CONTEXT_CHAIN_NAME
+            .scope(chain_name, transform.transform_pushed(self))
+            .await;
+        counter!("shotover_transform_total", 1, "transform" => transform_name);
```

Comment:
These metrics should probably be seperate to regular transform metrics `shotover_transform_pushed_total` or something.
My reasoning is that the values would be completely different and would mess up the regular data. e.g. latency would be wayyyy smaller.